phpunit.xml.dist

```diff
@@ -17,4 +17,8 @@
             <directory>src</directory>
         </whitelist>
     </filter>
+
+    <logging>
+        <log type="coverage-clover" target="build/coverage/clover.xml"/>
```

Comment:
IMO, it is better to use the command-line switch on Travis than forcing it in the default config